src/patternfly/components/Toggle/toggle.scss

```diff
@@ -0,0 +1,194 @@
+.pf-c-toggle {
+  --pf-c-toggle--PaddingTop: var(--pf-global--spacer--form-element);
+  --pf-c-toggle--PaddingRight: var(--pf-global--spacer--sm);
+  --pf-c-toggle--PaddingBottom: var(--pf-global--spacer--form-element);
+  --pf-c-toggle--PaddingLeft: var(--pf-global--spacer--sm);
+  --pf-c-toggle--FontSize: var(--pf-global--FontSize--md);
+  --pf-c-toggle--Color: var(--pf-global--Color--100);
+  --pf-c-toggle--LineHeight: var(--pf-global--LineHeight--md);
+  --pf-c-toggle--BackgroundColor: transparent;
+
+  // border base
+  --pf-c-toggle--before--BorderTopWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-toggle--before--BorderRightWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-toggle--before--BorderBottomWidth: 0;
+  --pf-c-toggle--before--BorderLeftWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-toggle--before--BorderTopColor: var(--pf-global--BorderColor--300);
+  --pf-c-toggle--before--BorderRightColor: var(--pf-global--BorderColor--300);
+  --pf-c-toggle--before--BorderBottomColor: transparent;
+  --pf-c-toggle--before--BorderLeftColor: var(--pf-global--BorderColor--300);
```

Comment:
I think this could just be `--pf-c-toggle--before--BorderWidth` and `--pf-c-toggle--before--BorderColor` as we assume that the `::before` is a simple border and the `::after` is fancier.

Reply:
It could... my only hesitation is with breaking changes where, if we change one of the border colors down the road and need to separate the vars (or just add a new var to define one of the sides), since `--pf-c-toggle--before--BorderColor` would no longer control all border colors, it could be considered breaking. Similar to how we separate hover/focus/active into separate vars, even if they're all the same. Using a common var sets us up for the role a var plays in a component to define something as no longer defining that thing. Though I don't feel strongly about it, we can't avoid some of those situations, though writing vars separately like this is one way to try and avoid that scenario.